pkgs/top-level/all-packages.nix

```diff
@@ -5919,6 +5919,8 @@ with pkgs;
 
   devpi-server = python3Packages.callPackage ../development/tools/devpi-server { };
 
+  dprint.plugins = recurseIntoAttrs (callPackage ../by-name/dp/dprint/plugins { });
```

Comment:
Hydra will not find the plugins when attached like this. Because recurseIntoAttrs isn't applied to dprint itself, it won't know to go one level deeper.

Instead, expose the plugins as dprint-plugins, and then re-expose them in the dprint package passthru.plugins.